app/jobs/split_pdf_job.rb

```diff
@@ -85,9 +118,9 @@ def save_pages(exam_template, partial_exams)
 
       group.access_repo do |repo|
         assignment_folder = exam_template.assignment.repository_folder
+        return if Admin.first.nil?
```

Comment:
Admin.empty? causes *** NoMethodError Exception: undefined method `empty?' for #<Class:0x00000007eb5dc8> so I used Admin.blank? instead.